web/src/app/modules/shared/components/smart/logs/logs.component.ts

```diff
@@ -78,7 +78,7 @@ export class LogsComponent
   }
 
   onSinceChange(sinceSelection: number): void {
-    this.selectedSince = sinceSelection;
+    this.selectedSince = +sinceSelection;
```

Comment:
Potentially redundant?

Reply:
I changed the `onSinceChange` handler to make this more clear. It now takes in the value as a string and converts it to a `number`. This is because the `(change)` $event.target.value will always be a string type.